modules/integration/src/test/scala/scala/cli/integration/RunEntrypointsTests.scala

```diff
@@ -0,0 +1,123 @@
+package scala.cli.integration
+
+import com.eed3si9n.expecty.Expecty.expect
+
+class RunEntrypointsTests extends munit.FunSuite {
+
+  val extraArgs = Seq(
+    "--scala=3.3.0-RC1-bin-20230120-d6cc101-NIGHTLY",
+    "--server=false",
+    "-r",
+    "sonatype:snapshots"
+  )
+
+  test("simple") {
+    val inputs = TestInputs(
+      os.rel / "Hello.scala" ->
```

Comment:
would `@entrypoint` also work with `.sc` files?

Reply:
It ought to, but it currently runs into TASTy version issues (that I need to look into)